scripts/terminalserver/terminalserver.jl

```diff
@@ -1,7 +1,12 @@
-
+import Pkg
```

Comment:
I feel like this will only be guaranteed to work if we start the process in our own env. The global one (which this process starts in implicitly, afaict) doesn't necessarily contain Pkg.

Reply:
I started implementing that, but there is a funny chicken-egg situation: if we can't import `Pkg` without having it in a project, then we cannot activate the project :) We can of course activate the project via command line flags, but overall this all seems a bit weird to me... It just generally seems that `Pkg.activate` is functionality that should be available always, completely regardless of any project loaded or not etc

Reply:
Mhm, yes. The whole Pkg/extensions story has made me pretty unhappy in that past.

This functionality is also available in `Base.set_active_project` (for 1.8 and up) or `Base.ACTIVE_PROJECT[] = ...`.

Reply:
Ok, new version pushed. But thinking about it, we should probably get rid of the `LOAD_PATH` modification and just put the VSCodeServer into the project we activate at the beginning...

Reply:
Yes, we should do that if we're having to mess with the active project.